scripts/prepare_netplugin_images.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# Prepare k8s netplugin image
+# * retrieves contiv/netplugin:$CONTIV_NETPLUGIN_VERSION from dockerhub
+#
+# or if $NETPLUGIN_BRANCH is set to an upstream branch name:
+# * downloads the branch from fork owner $NETPLUGIN_OWNER and build k8s netplugin
+#   image, export it as tarball
+
+set -euxo pipefail
+
+mkdir -p "$CONTIV_ARTIFACT_STAGING"
+
+#### ENSURE contiv/netplugin:<tag> exists #####
+
+if [ -z "${NETPLUGIN_BRANCH:-}" ]; then
+    echo "Trying to use dockerhub contiv/netplugin:${CONTIV_NETPLUGIN_VERSION}"
+    # ensure the image exists
+    http_rc=$(curl -s -w "%{http_code}" -o /dev/null  https://hub.docker.com/v2/repositories/contiv/netplugin/tags/${CONTIV_NETPLUGIN_VERSION}/)
```

Comment:
is this pulls the image, try using HEAD instead of GET with curl

Reply:
this pulls meta data of the image, not downloading it